lib/Mojolicious/Plugin/OpenAPI/Parameters.pm

```diff
@@ -41,7 +41,8 @@ sub _helper_build_schema_request {
     formData => sub {
       my $name  = shift;
       my $value = $req->body_params->every_param($name);
-      return $evaluated[@evaluated] = {exists => 1, value => $value} if @$value;
+      my $cnt   = @$value;
```

Comment:
This is a minor, but can you change `$cnt` to just `$n`? (Same below)

Reply:
Done